internal/build/phases_test.go

```diff
@@ -726,9 +721,6 @@ func testPhases(t *testing.T, when spec.G, it spec.S) {
 				h.AssertSliceNotContains(t, configProvider.ContainerConfig().Cmd, "-log-level", "debug")
 				h.AssertIncludeAllExpectedPatterns(t,
 					configProvider.ContainerConfig().Cmd,
-					[]string{"-layers", "/layers"},
-					[]string{"-app", "/workspace"},
-					[]string{"-platform", "/platform"},
 				)
```

Comment:
There are no checks happening here anymore. This assertion can be removed altogether.